src/Config.ts

```diff
@@ -287,6 +289,8 @@ let config: Config = {
   formingNetworkCycleThreshold: 30,
   maxResponseSize: 15 * 1024 * 1024, // 15MB
   enableDuplicateReceiptsCheck: true,
+  // cycle threshold for duplicate receipts overwrite check
+  duplicateReceiptsCheckActivationCycle: 0,
```

Comment:
We need to set this in env-specific configs. Mainnet / stagenet should be set based on the current cylce number (adding a buffer to acccount for deployment time)